Dockerfile

```diff
@@ -643,6 +643,11 @@ COPY --link --from=containerutil /build/ /
 COPY --link --from=vpnkit        /       /
 COPY --link --from=build         /build  /
 
+# usage:
+# > docker buildx bake bin-image
+FROM all AS bin-image
+COPY --link contrib/init /contrib/init
```

Comment:
Would it make sense to leave `bin-image` as binary-only, and have another target like `full-bin-image` that would put those contrib files in an extra layer?